cni.go

```diff
@@ -137,9 +137,19 @@ func (c *libcni) Load(opts ...Opt) error {
 func (c *libcni) Status() error {
 	c.RLock()
 	defer c.RUnlock()
-	if len(c.networks) < c.networkCount {
-		return ErrCNINotInitialized
+	if err := c.ready(); err != nil {
```

Comment:
here use the version of ready that does not "lock"

Reply:
> here use the version of ready that does not "lock"

These suggestions appeared to cause deadlocks. I am investigating